src/core/syscalls/business_logic_syscall_handler.rs

```diff
@@ -268,25 +268,30 @@ impl<'a, T: State + StateReader> BusinessLogicSyscallHandler<'a, T> {
         Ok(SyscallResponse { gas, body })
     }
 
+    fn constructor_entry_points(&self, contract_class: CompiledClass) -> Result<bool, StateError> {
```

Comment:
I'm not really sure of this function name. It sounds like it's gonna retrieve the constructor entrypoints. I would prefer something like `has_contructor_entry_points` or something like that. Also, returning a `Result<bool, Error>` looks a bit overkill. Maybe we can return just a bool.

Reply:
i cant return just a bool because of the `ok_or` call in the first match statement

Reply:
in exchange i have rename the function to a more suitable one.